celery/backends/base.py

```diff
@@ -460,7 +460,7 @@ def _get_result_meta(self, result,
                          state, traceback, request, format_date=True,
                          encode=False):
         if state in self.READY_STATES:
-            date_done = datetime.utcnow()
+            date_done = self.app.now()
```

Comment:
as this is understandable what the change is going to do, it is important to add or change existing tests to pass the test suit pass

Reply:
+1 -- https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow is deprecated in Python 3.12